nucleus/modelci/__init__.py

```diff
@@ -48,25 +51,46 @@ def __repr__(self):
     def __eq__(self, other):
         return self._connection == other._connection
 
-    def list_eval_functions(self) -> List[EvalFunction]:
+    @property
+    def eval_functions(self) -> AvailableEvalFunctions:
+        """List all available evaluation functions. ::
+
+        import nucleus
+        client = nucleus.NucleusClient("YOUR_SCALE_API_KEY")
+
+        eval_functions = client.modelci.eval_functions
```

Comment:
nit: add tab to this - I think it needs an extra tab space to show up as code in the docs